hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotLocalData.java

```diff
@@ -45,32 +51,33 @@ public abstract class OmSnapshotLocalData {
   // Whether SST is filtered
   private boolean isSSTFiltered;
 
-  // Map of Table to uncompacted SST file list on snapshot create
-  private Map<String, Set<String>> uncompactedSSTFileList;
-
   // Time of last compaction, in epoch milliseconds
   private long lastCompactionTime;
 
   // Whether the snapshot needs compaction
   private boolean needsCompaction;
 
-  // Map of version to compacted SST file list
-  // Map<version, Map<Table, sstFileList>>
-  private Map<Integer, Map<String, Set<String>>> compactedSSTFileList;
+  private UUID previousSnapshotId;
```

Comment:
> The snapshot meta should also track the previous snapshotId meta to know the version built from which snapshot and the corresponding version.

It occurred to be that we should have a defrag log somewhere to keep track of thing happening on each OM.